examples/frontend/containers/Layout.tsx

```diff
@@ -54,11 +60,15 @@ export default function Layout({ children, project = 'share' }) {
 
     const preSignOut = () => {
         localStorage.removeItem('contractAddress')
-        signOut()
         router.reload(window.location.origin + router.asPath)
+        signOut()
     }
 
     useEffect(switchProject, [router.asPath])
+    useEffect(switchProject, [])
```

Comment:
What's the difference between this and the line above it?

Reply:
We can remove that too. :-)